admin_manual/configuration_user/user_provisioning_api.rst

```diff
@@ -555,7 +555,44 @@ XML Output
       <element>testgroup</element>
     </data>
   </ocs>  
-  
+
+Resend the welcome email
+------------------------
+
+The request to this endpoint triggers the welcome email for this user again.
+
+**Syntax: ocs/v1.php/cloud/users/{userid}/welcome**
+
+* HTTP method: POST
+
+Status codes:
+
+* 100 - successful
+* 101 - email address not available
+* 102 - sending email failed
+
```

Comment:
When the user does not exist, you use `\OCP\API::RESPOND_UNAUTHORISED`, maybe we should change that to `RESPOND_NOT_FOUND`